custom_components/tahoma/sensor.py

```diff
@@ -301,6 +301,11 @@
         native_unit_of_measurement=TIME_SECONDS,
         entity_registry_enabled_default=False,
     ),
+    OverkizSensorDescription(
+        key="core:Memorized1PositionState",
+        name="MY position",
+        entity_registry_enabled_default=False,
```

Comment:
Could we add a default icon? Like the one I added here: https://github.com/iMicknl/ha-tahoma/pull/542.

This PR also implemented the same sensor by the way.

Reply:
since you mention the #542, maybe differentiate there between the 2 sensors? check mdi for 'signal' and a few good candidates emerge, or use wifi-strength for either of the 2.

sorry if this is off-topic here, let me know if you need a dedicated FR/PR for that

Reply:
I am not sure what you mean @Mariusthvdb, however for RSSI we don't set an icon. We set the         `device_class=sensor.DEVICE_CLASS_SIGNAL_STRENGTH` and front-end takes care of the icon.

Reply:
I thought about ⭐ like favorite, but also like you said about 💾 then I was confused and let the default one.

Reply:
> I am not sure what you mean @Mariusthvdb, however for RSSI we don't set an icon. We set the `device_class=sensor.DEVICE_CLASS_SIGNAL_STRENGTH` and front-end takes care of the icon.

I see. Its just that having the same icon for 2 different (discrete and numerical) sensors can be a bit visually confusing. Since they both pertain the RSII level, maybe not to worry.
and as before, an icon template is easily made.